tests/test_database.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+sys.path.append(f'{os.path.dirname(__file__)}/..')
+
+import modules.database as database
+
+# users table tests
+
+def test_add_user():
+    num_users = 1000
+    for i in range(num_users):
+        database.users.add(f'user{i}')
+    
+    assert len(database.users.all()) == num_users
+
+def test_delete_user():
+    database.users.add("user")
+    user_id = database.users.all()[0]
```

Comment:
Make user object instead of just passing a string to this methods.
Also if you are retrieving the id from this list (variable name says so), make it take the property of "Id" instead just the user object
```suggestion
    database.users.add(mockUsers[0])
    user_id = database.users.all()[0].Id
```

Reply:
Partial agree, Partial disagree.
List of user objects instead of singular user string, i agree.
user_id fetching should be discarded. We want to index on the user_id and not on their name, and only fetch the requested user and not *all* users and then grab a single entry.